redisinsight/ui/src/pages/browser/components/bulk-actions/BulkActionsInfo/BulkActionsInfo.tsx

```diff
@@ -38,77 +42,47 @@ const BulkActionsInfo = (props: Props) => {
+            {subTitle}
+          </BulkActionsTitle>
+        )}
+        <Row justify="start" align="center" gap="s">
           {filter && (
-            <div
-              className={styles.filter}
-              data-testid="bulk-actions-info-filter"
-            >
-              <span style={{ paddingRight: 6 }}>Key type:</span>
-              <GroupBadge type={filter} className={styles.badge} />
-            </div>
+            <BulkActionsInfoFilter data-testid="bulk-actions-info-filter">
+              <div>Key type:</div>
+              <GroupBadge type={filter} />
+            </BulkActionsInfoFilter>
           )}
           {search && (
-            <div
-              className={styles.search}
-              data-testid="bulk-actions-info-search"
-            >
+            <BulkActionsInfoFilter data-testid="bulk-actions-info-search">
               Pattern:
-              <span className={styles.match}>{` ${search}`}</span>
-            </div>
+              <BulkActionsInfoSearch color="subdued">
+                {' '}
+                {search}
+              </BulkActionsInfoSearch>
+            </BulkActionsInfoFilter>
           )}
-        </Text>
-        {!isUndefined(status) && !isProcessedBulkAction(status) && (
-          <Text
-            color="subdued"
-            className={styles.progress}
-            data-testid="bulk-status-progress"
-          >
-            In progress:
-            <span>{` ${getApproximatePercentage(total, scanned)}`}</span>
-          </Text>
-        )}
-        {status === BulkActionsStatus.Aborted && (
-          <Text
-            color="danger"
-            className={styles.progress}
-            data-testid="bulk-status-stopped"
-          >
-            Stopped: {getApproximatePercentage(total, scanned)}
-          </Text>
-        )}
-        {status === BulkActionsStatus.Completed && (
-          <Text
-            className={cx(styles.progress, styles.progressCompleted)}
-            data-testid="bulk-status-completed"
-          >
-            Action completed
-          </Text>
-        )}
-        {status === BulkActionsStatus.Disconnected && (
-          <Text
-            color="danger"
-            className={styles.progress}
-            data-testid="bulk-status-disconnected"
-          >
-            Connection Lost: {getApproximatePercentage(total, scanned)}
-          </Text>
-        )}
-      </div>
-      <Divider colorVariable="separatorColor" className={styles.divider} />
+        </Row>
+      </Col>
+      <Divider colorVariable="separatorColor" />
       {loading && (
-        <div className={styles.progressLine} data-testid="progress-line">
+        <BulkActionsProgressLine data-testid="progress-line">
           <div style={{ width: `${(total ? scanned / total : 0) * 100}%` }} />
-        </div>
+        </BulkActionsProgressLine>
       )}
-      <div className={styles.children}>{children}</div>
-    </div>
+      <div>{children}</div>
```

Comment:
I've noticed that the text we show in the info tooltip on the "**Delete Keys**" tab is slightly misaligned. However, we don't have a reference design, so I'm not sure what the expected alignment is - left or justified.

<img width="552" height="277" alt="Image" src="https://github.com/user-attachments/assets/a7af638c-1a38-48b3-9b79-94867d732641" />